charts/cyclecloud/templates/Deployment.yaml

```diff
@@ -70,13 +70,14 @@ spec:
         image: "{{ .Values.cycle.containerImage }}"
         resources:
           requests:
-            memory: "10Gi"
+            memory: "20Gi"
           limits:
-            memory: "10Gi"
+            memory: "20Gi"
+            cpu: "8"
         command: ["/bin/bash"]
         securityContext:
           allowPrivilegeEscalation: false  
-        args: ["-c", "mkdir -p /mnt/cycle_server/data /mnt/cycle_server/logs /mnt/cycle_server/.ssh && chmod 700 /mnt/cycle_server/.ssh && chmod +x scripts/run_cyclecloud.sh && ./scripts/run_cyclecloud.sh"]
+        args: ["-c", "mkdir -p /mnt/cycle_server/data /mnt/cycle_server/logs /mnt/cycle_server/.ssh && sed -i \"s/webServerMaxHeapSize\\=.*/webServerMaxHeapSize\\=$JAVA_HEAP_SIZE/\" /opt/cycle_server/config/cycle_server.properties && chmod 700 /mnt/cycle_server/.ssh && chmod +x scripts/run_cyclecloud.sh && ./scripts/run_cyclecloud.sh"]
```

Comment:
Does this actually work?
I was thinking:
sed -i \"s/webServerMaxHeapSize\\=.*/webServerMaxHeapSize\\={{ .Values.cycle.webServerMaxHeapSize }}/\"